api/query.go

```diff
@@ -11,24 +11,47 @@ import (
 	"github.com/equinor/vds-slice/internal/vds"
 )
 
-type MetadataRequest struct {
+type BaseRequest struct {
+	// The blob path to a vds in form: container/subpath
 	Vds string `json:"vds" binding:"required"`
+	// A valid sas-token with read access to the container specified in Vds
 	Sas string `json:"sas" binding:"required"`
+}
+
+type MetadataRequest struct {
+	BaseRequest
 } //@name MetadataRequest
 
 type FenceRequest struct {
-	Vds              string      `json:"vds"               binding:"required"`
-	CoordinateSystem string      `json:"coordinate_system" binding:"required"`
-	Fence            [][]float32 `json:"coordinates"       binding:"required"`
-	Interpolation    string      `json:"interpolation"`
-	Sas              string      `json:"sas"               binding:"required"`
-}
+	BaseRequest
+	// Coordinate system for the requested fence
+	// Supported options are:
+	// ilxl : inline, crossline pairs
+	// ijk  : Coordinates are given as in 0-indexed system, where the first
+	//        line in each direction is 0 and the last is number-of-lines - 1.
+	// cdp  : Coordinates are given as cdpx/cdpy pairs. In the original SEGY
+	//        this would correspond to the cdpx and cdpy fields in the
+	//        trace-headers.
+	CoordinateSystem string `json:"coordinate_system" binding:"required"`
+
+	// A list of (x, y) points in the coordinate system specified in
+	// coordinate_system.
+	Fence [][]float32 `json:"coordinates" binding:"required"`
```

Comment:
Yes, we can create prototypes with more sensible values. I haven't done it for any of the others because the standard ones was ok ish. Here it is horrible, I do agree. I'll make an issue for it and we can do another PR adding prototypes for all exposed types

Reply:
Or maybe I misunderstand swag proto's examples. Alternatively we can add examples to the field tags

Reply:
I made an issue for it #31